website/docs/r/workspace.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_workspoce"
+sidebar_current: "docs-tfe-workspace"
+description: |-
+  Workspaces represent running infrastructure managed by Terraform.
+---
+
+# tfe_workspoce
+
+Provides a workspace resource.
```

Comment:
maybe this'd be better as: `Manages a Workspace in Terraform Enterprise. Workspaces represent running infrastructure managed by Terraform`